### PR TITLE
Add hex metadata to app.src

### PR DESCRIPTION
**Pull request by nalundgaard**

### CODE REVIEW

File: src/erl_cache.app.src [hunk 1]
```diff
@@ -1,13 +1,15 @@
 {application, erl_cache,
  [
-  {description, ""},
-  {vsn, git},
-  {registered, []},
+  {description, "A generic in-node caching library with memoization support"},
+  {vsn, "1.7.1-alertlogic"},
```
Comment (nalundgaard) [inline src/erl_cache.app.src]: IDK if this is the best to do or if it's preferable to support the "git" thing still.

Comment (motobob) [reply src/erl_cache.app.src]: `rebar3 hex publish` supports git tags as longs as they are annotated. this is unnecessary.

**Review by motobob — DISMISSED** [review]

File: src/erl_cache.app.src (hunk 1 above)
Comment (motobob) [inline src/erl_cache.app.src]: "git" is fine - just checked

File: src/erl_cache.app.src [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
   {applications, [
                   kernel,
                   stdlib,
                   crypto
                  ]},
   {mod, { erl_cache_app, []}},
-  {env, []}
+  {env, []},
+  {maintainers, ["Nicholas Lundgaard", "Evgeny Bob"]},
```
Comment (motobob) [inline src/erl_cache.app.src]: maintainer declared deprecated

**Comment by nalundgaard** [comment]
Still failing because I need to fork and publish decorator_pt first. Yay.

**Pull request merged**